docs/data/joy/components/modal/modal.md

```diff
@@ -75,14 +75,30 @@ The possible values are:
 
 {{"demo": "CloseModal.js"}}
 
-### Modal Dialog
+### Modal dialog
 
 To create a modal dialog, renders the `ModalDialog` component inside the `Modal`.
 
 It will apply spacing to the elements that have `aria-labelledby` or `aria-describedby` attribute.
```

Comment:
Does "It" refer to the Modal or the Modal Dialog?

Reply:
I believe it is referring to Modal Dialog!

Reply:
Yes, "it" refers to the Modal Dialog.

Reply:
How about this for brevity + clarity?

```suggestion
The Dialog will apply spacing to the elements that have `aria-labelledby` or `aria-describedby` attribute.
```